package.json

```diff
@@ -95,8 +94,8 @@
     "sanitize-html": "^1.18.2",
     "serve-favicon": "^2.3.2",
     "socketio": "^1.0.0",
-    "socketio-file-upload": "^0.6.2",
-    "strip-bom": "^3.0.0",
+    "socketio-file-upload": "^0.7.0",
```

Comment:
this can be removed
```suggestion
```

Reply:
used in src/sockets/clipboard/index.js und upload.js enthalten